flask_app/views.py

```diff
@@ -24,7 +24,9 @@ def predict():
         start_time = time.time()
 
         model = model_store.get('stargan')
-
+        
+        logging.getLogger().AddINFO(str(req))
```

Comment:
change to `logging.debug`

Reply:
Logging out the base64 encoded images is also kind of messy. See whether there are other properties in the request that we can log?

Reply:
The request sends out 3 pieces of data, 2 image files which are the source and reference image the GAN requires, as well as a string for the gender of the image.

Reply:
I have added the requests header to logging. An example of the request header is below:

Host: 123.456.78.910:1112
User-Agent: python-requests/2.25.1
Accept-Encoding: gzip, deflate
Accept: */*
Connection: keep-alive
Content-Length: 22541
Content-Type: application/json